src/clappr-dash-shaka-playback.js

```diff
@@ -261,6 +294,9 @@ class DashShakaPlayback extends HTML5Video {
     this._options.shakaConfiguration && this._player.configure(this._options.shakaConfiguration)
     this._options.shakaOnBeforeLoad && this._options.shakaOnBeforeLoad(this._player)
 
+    const preferredAudioLanguage = this._player.getConfiguration().preferredAudioLanguage
```

Comment:
Isn't this something that can be overridden via `shakaConfiguration` or `_options`?

Reply:
just by `_options` not, but it is actually setup by the `this._player.configure(this._options.shakaConfiguration)` call, two lines above ^

now we need to initialize the state of `_activeAudioLanguage` because the initial language will be the one set in `preferredAudioLanguage` if it is set there. 

otherwise me start out with saying language A is active because it's first in the list, but actually language B is because it has been configured as preferred.

not sure i got the question right, but does that answer your question actually? :)

Reply:
I think so, this `this._options.shakaConfiguration.preferredAudioLanguage` is equivalent to this `this._player.getConfiguration().preferredAudioLanguage`.